### ISSUE TITLE
Bug: Own STUN/TURN server config file issue

### ISSUE THREAD

**Issue by mochman** [schlagmichdoch/PairDrop#63]
**Describe the bug**
When using the RTC_CONFIG env variable, I can't connect to anyone to send files.
It looks like it stems from where the file is read in index.js.  It doesn't parse the JSON, so it has a bunch of escaped characters it doesn't know what to do with.
![badrtc](https://user-images.githubusercontent.com/19363185/222991563-a8fb18d5-a11e-4fc1-ba9f-08ab7a34771b.png)

This is not how it looks when no RTC_CONFIG var is set.
![nortc](https://user-images.githubusercontent.com/19363185/222991587-ff2a5ff7-8eb4-4860-b84d-129212d08ba1.png)

I was able to fix the issue by wrapping the line:
https://github.com/schlagmichdoch/PairDrop/blob/b2f6a75c991a212566fb3d5763236bb1d176ee32/index.js#L55

with:
```
? JSON.parse(fs.readFileSync(process.env.RTC_CONFIG, 'utf8'))
```
Using that new index.js with the RTC_CONFIG var, I get the expected results.
![goodrtc](https://user-images.githubusercontent.com/19363185/222991742-03f2716c-bedd-4c14-b7a4-314071dc284e.png)

I was able to connect to my own STUN server and see other people on my network to send files to.

Extra Troubleshooting Steps:
I tried using the already made example rtc_config by setting `RTC_CONFIG=/app/pairdrop/rtc_config_example.json`.  It still had all the escaped newlines and quotes like in the top screenshot.  So I don't think the issue was my rtc_config.json formatting.

**To Reproduce**
1. Set RTC_CONFIG env var to a json file with the appropriate config data.

**Expected behavior**
Parse the JSON data.

**Screenshots**
See above

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome
 - Version: 22

**Self-Hosted**
Yes

**Self-Hosted Setup**
Proxy: Nginx
Deployment: docker-compose

**Comment by mochman** [comment]
I can submit a pull request with the change if you want me to.

**Comment by schlagmichdoch** [comment]
Thanks @mochman for the comprehensive issue description! What a stupid mistake. I don't know why this worked when I tested it 🤷 

Thanks for your PR too! I appreciate all contributions! I got notified about this issues some hours ago already [here](https://github.com/schlagmichdoch/PairDrop/commit/66359da2cac78de868a17a90942f117dd8cdaac1#r103132889) and have only seen your comment after already pushing the changes. This is fixed now.
